apps/studio/components/layouts/AppLayout/EnableBranchingButton/EnableBranchingModal.tsx

```diff
@@ -123,9 +114,9 @@ const EnableBranchingModal = () => {
     defaultValues: { branchName: '' },
   })
 
-  const isLoading = isLoadingRepositories
-  const isError = isErrorRepositories || isErrorUpgradeEligibility
-  const isSuccess = isSuccessRepositories && isSuccessUpgradeEligibility
+  const isLoading = isLoadingConnections || isLoadingUpgradeEligibility
+  const isError = isErrorConnections || isErrorUpgradeEligibility
```

Comment:
i just realised we're not using the error, isLoading, isError, isSuccess from `connections` - do we need to?

Reply:
Yeah I think it was a mistake that they weren't included before. Most of the time the user doesn't see the loading state anyway since it's loaded in the background, but better safe than sorry if they manage to open the modal quick!